tests/test_queries.py

```diff
@@ -112,6 +112,19 @@ def test(value):
     assert not query({'': None})
 
 
+def test_custom_with_params():
+    def test(value, minimum, maximum):
+        return minimum <= value <= maximum
+
+    query = where('val').test(test, 1, 10)
+
+    assert query({'val': 5})
+    assert not query({'val': 0})
+    assert not query({'val': 11})
+    assert not query({'val': '44'})
```

Comment:
You're using a string for `44`, not an int. This breaks the tests for Python3 and PyPy3 (see https://travis-ci.org/msiemens/tinydb/builds/68895961).